src/helix/core/HelixSprite.hx

```diff
@@ -142,6 +147,35 @@ class HelixSprite extends FlxSprite
         }
         return y;
     }
+    
+    private function resetAcceleration():Void
+    {
+        this.acceleration.set();
+        this.angularVelocity = 0;
+    }
+
+    public function getFlxKeyArray(keys:Array<String>):Array<FlxKey>
+    {
+        var flxKeyArray = new Array<FlxKey>();
+
+        for (key in keys) {
+            var flxKey:Null<FlxKey> = FlxKey.fromString(key);
+
+            if (flxKey != null) {
+                flxKeyArray.push(flxKey);
+            }
+        }
+        return flxKeyArray;
+    }
+
+    private function processKeybinds():Void
+    {
+        for (flxKeyArray in this.keybindMap.keys()) {
+            if (FlxG.keys.anyPressed(flxKeyArray)) {

```

Comment:
Instead of this, you can just pass `FlxG.keyboard.getIsDown()` ([docs](http://api.haxeflixel.com/flixel/input/FlxKeyManager.html#getIsDown)). You may need to figure out how to expose this as just a simple array of keys to the caller, so users can use something like `if keys.indexOf(FlxKey.A) > -1)` or something like that.

This code definitely  needs to move into `update` and we need to do this if we have a callback specified (non-null).